src/pages/authors/index.astro

```diff
@@ -4,14 +4,15 @@ import Info from "~/assets/svgs/info.astro";
 import AuthorCard from "~/components/cards/authorCard.astro";
 import ListLayout from "~/layouts/list.astro";
 import { authorsHandler } from "~/lib/handlers/authors";
+import type {Author} from "~/lib/types"
 import { getMeta } from "~/lib/utils/getMeta";
 const entry = await getEntry("views", "authors");
 
 if (!entry) {
   return Astro.redirect("/404");
 }
 
-const authors = authorsHandler.allAuthors();
+const authors:Author[] = authorsHandler.allAuthors();
```

Comment:
In this section, there's no need to define the types manually; Astro will generate them automatically